django/bossspatialdb/views.py

```diff
@@ -79,13 +79,22 @@ def get(self, request, collection, experiment, channel, resolution, x_range, y_r
         else:
             iso = False
 
+        #Default access_mode to cache on the server side in case it is not explicitly defined in an API call. 
+        access_mode = "cache"
+        #Translation from no-cache boolean param to access_mode param for backwards compatability. 
         if "no-cache" in request.query_params:
             if request.query_params["no-cache"].lower() == "true":
-                no_cache = True
+                access_mode = "no_cache"
+            elif request.query_params["no-cache"].lower() == "false":
+                access_mode = "cache"
+                
+        if "access_mode" in request.query_params:
```

Comment:
If both `no-cache` and `access_mode` are specified the new parameter overrides the old parameter?

Reply:
Changed the code to raise an error if both no-cache and access_mode parameters are specified

Reply:
@movestill just pushed